test/test-classTetrisMoves.js

```diff
@@ -0,0 +1,312 @@
+        stubForGetRandomTypeOfTetromino.returns("I")
+        gameTetris.tick()
+        let expectedFieldAddI = tetris.buildField(`
+            --------
+            --------
+            --------
+            --------
+            --X-----
+            XXXX-XXX
+        `)
+        assert.deepEqual(gameTetris.field.field, expectedFieldAddI)
+        let expectedNewTetrominoISecomd = tetris.getCoordinates(`
+            --XXXX--
+            --------
+            --------
+            --------
+            --------
+            --------
+        `)
+        assert.deepEqual(gameTetris.tetromino.coordinates, expectedNewTetrominoISecomd)
+        gameTetris.tick()
+        gameTetris.rotate()
+        gameTetris.moveRight()
+        gameTetris.moveRight()
+        gameTetris.tick()
+        let expectedTicksRotateMovesRightI = tetris.getCoordinates(`
+            --------
+            -----X--
+            -----X--
+            -----X--
+            -----X--
+            --------
+        `)
+        assert.deepEqual(gameTetris.tetromino.coordinates, expectedTicksRotateMovesRightI)
+        stubForGetRandomTypeOfTetromino.returns("O")
+        gameTetris.tick()
+        let expectedFieldAddISecond = tetris.buildField(`
+            --------
+            -----X--
+            -----X--
+            -----X--
+            --X--X--
+            XXXX-XXX
+        `)
+        assert.deepEqual(gameTetris.field.field, expectedFieldAddISecond)
+        let expectedNewTetrominoOSecond = tetris.getCoordinates(`
+            ---XX---
+            ---XX---
+            --------
+            --------
+            --------
+            --------
+        `)
+        assert.deepEqual(gameTetris.tetromino.coordinates, expectedNewTetrominoOSecond)
+        gameTetris.moveRight()
+        gameTetris.moveRight()
+        gameTetris.tick()
+        gameTetris.tick()
+        gameTetris.tick()
+        let expectedMovesRightTicksO = tetris.getCoordinates(`
+            --------
+            --------
+            --------
+            ---XX---
+            ---XX---
+            --------
+        `)
+        assert.deepEqual(gameTetris.tetromino.coordinates, expectedMovesRightTicksO)
+        stubForGetRandomTypeOfTetromino.returns("I")
+        gameTetris.tick()
+        let expectedFieldAddOSecond = tetris.buildField(`
+            --------
+            -----X--
+            -----X--
+            ---XXX--
+            --XXXX--
+            XXXX-XXX
+        `)
+        assert.deepEqual(gameTetris.field.field, expectedFieldAddOSecond)
+        gameTetris.tick()
+        let expectedFieldAddOThird = tetris.buildField(`
+            --XXXX--
+            -----X--
+            -----X--
+            ---XXX--
+            --XXXX--
+            XXXX-XXX
+        `)
+        assert.deepEqual(gameTetris.field.field, expectedFieldAddOThird)
+        gameTetris.tick()
+        let expectedStopGame = tetris.buildField(`
```

Comment:
you are not checking that the game is stopped, you just checking the field.
Please add assert that callback is called